EvotoApi/App_Start/Startup.Auth.cs

```diff
@@ -27,6 +27,10 @@ public void ConfigureAuth(IAppBuilder app)
             {
                 AuthenticationType = DefaultAuthenticationTypes.ApplicationCookie,
                 LoginPath = new PathString("/manage/login"),
+                SlidingExpiration = true,
+                CookieHttpOnly = true,
```

Comment:
Will this break things when we're using production servers, which do use https? Maybe #if DEBUG or something to fix.